src/moov/trak/mdia/minf/stbl/stsd/chnl.rs

```diff
@@ -0,0 +1,396 @@
+use crate::*;
+
+// ISO/IEC 23091-3 Audio Channel Position
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[repr(u8)]
+pub enum AudioChannelPosition {
```

Comment:
Are these intended to match ISO/IEC 23091-3:2018 Table 2? Looks like some of the name positions are from IEC 62574.

Also, some of the values (36-44, if including Amendment 1) are missing.

Reply:
I will check.